primitive-types/impls/codec/Cargo.toml

```diff
@@ -1,13 +1,13 @@
 [package]
 name = "impl-codec"
-version = "0.2.0"
+version = "0.3.0"
 authors = ["Parity Technologies <admin@parity.io>"]
 license = "Apache-2.0/MIT"
 homepage = "https://github.com/paritytech/parity-common"
 description = "Parity Codec serialization support for uint and fixed hash."
 
 [dependencies]
-parity-codec = { version = "3.0", default-features = false }
+parity-codec = { version = "4.0", default-features = false }
```

Comment:
Did the crate name stay the same?

Reply:
Yes for now. We currently have 4.0 version of `parity-codec`, it is renamed to `parity-scale-codec` and will be released as `1.0.0` in the future. So we need to break everything twice.